packages/l10n/utils.js

```diff
@@ -0,0 +1,41 @@
+export const mapLocaleToIntlLocale = locale => {
+  if (locale.startsWith('de')) return 'de';
+  if (locale.startsWith('es')) return 'es';
+  if (locale.startsWith('fr')) return 'fr-FR';
+  if (locale === 'zh-CN') return 'zh-CN';
+  return 'en';
+};
+
+const supportedLocales = ['en', 'de', 'es', 'fr-FR', 'zh-CN'];
```

Comment:
Wondering if we should share this somehow in a pkg?

Reply:
I thought about it but each package uses the info a bit differently, so I would prefer to keep it co-located to each package for now.

Reply:
I just felt it being a bit "dangerous" given things get out of sync across packages but lets see how it goes.